index.js

```diff
@@ -1123,6 +1045,36 @@ instance.prototype.action = function(action) {
 				'scene-name': sceneName
 			});
 			break;
+		case 'toggle_filter_item':
+				let visible = true
+				let sceneName = action.options.scene && action.options.scene != "" ? action.options.scene : null
+
```

Comment:
I believe the above 2 variables are already used and defined in the toggle_scene_item action

Reply:
Sorry i don't understand the problem here :/